doc/source/_static/custom.css

```diff
@@ -0,0 +1,3 @@
+div.section a span {

```

Comment:
For some reason (and I think specifically with the sphinx RTD theme) the clickable links within examples don't show up as the "link" color, but just as regular text, which makes it hard to spot as a user. This forces them to do so.